app/workers/bgs/submit_form674_job.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'bgs/form674'
+
+module BGS
+  class SubmitForm674Job
+    class Invalid674Claim < StandardError; end
+    include Sidekiq::Worker
+    include SentryLogging
+
+    # we do individual service retries in lib/bgs/service.rb
+    sidekiq_options retry: false
+
+    def perform(user_uuid, saved_claim_id, vet_info)
+      user = User.find(user_uuid)
+      claim_data = valid_claim_data(saved_claim_id, vet_info)
+
+      BGS::Form674.new(user).submit(claim_data)
+    rescue
```

Comment:
Should we eat all errors here?

Reply:
I think so. We shoot off a lot of requests (like 20) to our integration partner BGS from this job. I can't get a sense of consistency from BGS services in terms of exceptions. We handle retry logic further down stream given some nuances between services. If an error bubbles up to here, failure was catastrophic.